src/Services/FHIR/FhirAllergyIntoleranceService.php

```diff
@@ -47,7 +47,8 @@ public function __construct()
     protected function loadSearchParameters()
     {
         return  [
-            'patient' => ['patient.uuid']
+            'patient' => ['lists.pid'],
+            '_id' => ['lists.id']
```

Comment:
Are we using uuid or id's here?

Reply:
The way code is written in AllergyIntoleranceService.php, the uuid is taken and then  converted to respective id and replaced in the same field.

```$puuidBytes = UuidRegistry::uuidToBytes($search['lists.pid']);
            $search['lists.pid'] = $this->getIdByUuid($puuidBytes, self::PATIENT_TABLE, "pid");```

Reply:
hi @vishnuyar , thanks for pointing that out. this PR looks good and bringing in. thanks for the fixes/improvements! -brady